jazz-build-module/scm-module.groovy

```diff
@@ -28,13 +27,15 @@ def setRepoEndpoints(){
         scm_user_services_clone_url = "${scm_protocol}${scm_config.REPOSITORY.BASE_URL}/${scm_config.REPOSITORY.REPO_BASE_SERVICES}/"
         scm_branch_permission_api_endpoint = "${scm_protocol}${scm_config.REPOSITORY.BASE_URL}/rest/branch-permissions/2.0/projects/${scm_config.REPOSITORY.REPO_BASE_SERVICES}/repos/"
         serviceonboarding_repo	= "${scm_protocol}${scm_config.REPOSITORY.BASE_URL}/${scm_config.REPOSITORY.REPO_BASE_PLATFORM}/service-onboarding-build-pack.git"
-    }else if(scm_config.SCM.TYPE == "bitbucket"){
+		scm_webhook_target_url 	= "${scm_protocol}${scm_config.JENKINS.USERNAME}:${scm_config.JENKINS.JENKINS_PASSWORD}@${JenkinsLocationConfiguration.get().getUrl().split('/')[2]}/project/Gitlab-Trigger-Job"
```

Comment:
We cannot have JenkinsLocationConfiguration come into SCM Module as we are no longer separating concerns.  The url value can be passed into the method "addWebhook" as a variable

Reply:
In cooperated the review comment